api/utils/sharing/sharing.go

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package v1alpha1
+package sharing
```

Comment:
So these utils are no longer versioned? Is that because we don't expect them to be public?

Reply:
I take it back -- the apis that use this are still versioned, so as long as we only change this in conjunction with a n bump on those API versions, we are OK.